src/components/ebay-bar-chart/component.js

```diff
@@ -0,0 +1,346 @@
+import Highcharts from 'highcharts';
+import accessibility from 'highcharts/modules/accessibility';
+import patternFill from 'highcharts/modules/pattern-fill';
+
+import {
+    chartFontFamily,
+    backgroundColor,
+    gridColor,
+    labelsColor,
+    legendColor,
+    legendInactiveColor,
+    legendHoverColor,
+    tooltipBackgroundColor,
+    tooltipShadows,
+    setSeriesColors,
+} from '../../common/charts/shared';
+
+import { ebayLegend } from '../../common/charts/legend';
+import { eBayColumns } from '../../common/charts/bar-chart';
+
+export default class {
+    onMount() {
+        this._initializeHighchartsExtensions();
+        this._setupCharts();
```

Comment:
If this component rerenders, will highcharts still work? If I change input for example.